barter-data/src/provider/databento/mod.rs

```diff
@@ -0,0 +1,33 @@
+use barter_instrument::Side;
+use databento::dbn::Side as DbSide;
+use serde::{Deserialize, Serialize};
+
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Deserialize, Serialize)]
+pub enum DatabentoSide {
```

Comment:
By the looks of this the Barter Side should deserialize and serialize appropriately